### PR TITLE
update-all-frontend-tests-name

### PR DESCRIPTION
**Pull request by selvaraj-sembulingam**

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy_tests/test_ivy/test_frontends/test_jax/test_devicearray.py [hunk 1]
```diff
@@ -1088,7 +1088,7 @@ def test_jax_devicearray__rxor_(
         available_dtypes=st.one_of(st.just(("bool",)), helpers.get_dtypes("integer")),
     ),
 )
-def test_jax_devicearray__invert_(
+def test_jax__invert_(
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_devicearray.py:1091]: I think you have missed some underscores in the special methods tests. For example, here the name of the function is `__invert__` so the test should be `test_jax___invert__`.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py [hunk 2]
```diff
@@ -1294,7 +1294,7 @@ def test_numpy_ndarray_fill(
         available_dtypes=helpers.get_dtypes("numeric"), num_arrays=2
     ),
 )
-def test_numpy_instance_add__(
+def test_numpy_add__(
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:1297]: Same thing in this file. The easiest way to not get confused is to copy the name from `method_name` above.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 3]
```diff
@@ -93,7 +93,7 @@ def _get_clip_inputs(draw):
         available_dtypes=helpers.get_dtypes("valid", prune_function=False)
     ).filter(lambda x: "bfloat16" not in x[0]),
 )
-def test_paddle_instance_property_device(
+def test_paddle_property_device(
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:96]: "property" should not be included in the test names.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_norms.py [hunk 4]
```diff
@@ -81,7 +81,7 @@ def test_torch_batch_norm(
     data=_instance_and_batch_norm_helper(min_num_dims=3, min_dim_size=2),
     use_input_stats=st.booleans(),
 )
-def test_torch_instance_norm(
+def test_torch___norm(
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_norms.py:84]: This was correct the way it was before.

**Review by AnnaTz — CHANGES_REQUESTED** [review]
Just a few cases that slipped under the radar.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py (hunk 2 above)
Comment (selvaraj-sembulingam) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py]: Done

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py (hunk 3 above)
Comment (selvaraj-sembulingam) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py]: Done

File: ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_norms.py (hunk 4 above)
Comment (selvaraj-sembulingam) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_norms.py]: Done

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 5]
```diff
@@ -9568,7 +9568,7 @@ def test_torch_instance_addr_(
         allow_inf=False,
     ),
 )
-def test_torch_special_eq_(
+def test_torch___eq___(
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:9571]: This is testing `eq_`, not `__eq__`. The name of the test was wrong.

**Review by AnnaTz — CHANGES_REQUESTED** [review]
There are also `__pycache__` files that should be removed before merging.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py (hunk 5 above)
Comment (selvaraj-sembulingam) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py]: Done

**Review by AnnaTz — APPROVED** [review]
Thanks a lot!

**Pull request merged**